banking-client-api/src/main/java/api/response/TransferResponse.java

```diff
@@ -6,7 +6,9 @@
 @AllArgsConstructor
 @Getter
 public class TransferResponse {
+    String tx_id;
```

Comment:
@juseon-park 
```suggestion
    String txId;
```

언더바보다는 카멜케이스를 씁시다!

Reply:
@juseon-park 
`@JsonProperty`